packages/ui-extensions/src/surfaces/checkout-preact.ts

```diff
@@ -0,0 +1 @@
+export * from './checkout/preact';
```

Comment:
Do we need this one? We can just do 
```
pkg.entry({
    name: 'checkout/preact',
    root: './src/surfaces/checkout/index.ts',
  });
```

Reply:
The types are in

```
      "checkout/preact": [
        "./build/ts/surfaces/checkout-preact.d.ts"
      ],
```

would it be OK to nest them like

```
      "checkout/preact": [
        "./build/ts/surfaces/checkout/preact.d.ts"
      ],
```

I'm just wondering if that would conflict with `./build/ts/surfaces/checkout.d.ts` 🤔 I guess not.